### DISCUSSION TITLE
Zoom rate in my current project is not fitting correclty with screen

### DISCUSSION THREAD

**Discussion by BennaceurHichem** [tailwindlabs/tailwindcss#7300] [Help]
I had a project where I worked with Nuxt.js and tailwind, I had a problem of screen content positioning, the screen now shows the layouts in such a zoomed version by default(100% zoom rate), and the best zoom rate that cover correctly the different layouts and components of the screen is 80% 
**This is a 100% rate that didn't work properly for me**
![image](https://user-images.githubusercontent.com/31354135/152129370-614eb151-2377-438d-b6c3-eb69f79f3911.png)

**This is 80% which make layouts and components fits properly with the screen**
![image](https://user-images.githubusercontent.com/31354135/152129489-616b69f4-0e2b-497f-a379-24e8767e96e8.png)

I'm not sure the reason of this and it seems difficult for me to debug it
* I didn't add any responsive design config in tailwindcss 
* I added this by default in `nuxt.config.js` : `      { name: 'viewport', content: 'width=device-width, initial-scale=1' },`

should I focus on making the zoom rate 80%, or is there any possible reason for this that I need to fix? 
does anyone face this issue?

Thanks.

**Comment by jairsemblantes01** [comment]
tengo el mismo problema
